[2022-07-26 13:11:55,817] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag.task3 manual__2022-07-26T13:11:53.156828+00:00 [queued]>
[2022-07-26 13:11:55,853] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag.task3 manual__2022-07-26T13:11:53.156828+00:00 [queued]>
[2022-07-26 13:11:55,854] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-26 13:11:55,855] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-07-26 13:11:55,855] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-26 13:11:55,880] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): task3> on 2022-07-26 13:11:53.156828+00:00
[2022-07-26 13:11:55,886] {standard_task_runner.py:52} INFO - Started process 6087 to run task
[2022-07-26 13:11:55,891] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'task3', 'manual__2022-07-26T13:11:53.156828+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpl5io0bnh', '--error-file', '/tmp/tmpbb3m40kf']
[2022-07-26 13:11:55,893] {standard_task_runner.py:80} INFO - Job 136: Subtask task3
[2022-07-26 13:11:56,154] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag.task3 manual__2022-07-26T13:11:53.156828+00:00 [running]> on host 24820bb59fd1
[2022-07-26 13:11:56,337] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag
AIRFLOW_CTX_TASK_ID=task3
AIRFLOW_CTX_EXECUTION_DATE=2022-07-26T13:11:53.156828+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-26T13:11:53.156828+00:00
[2022-07-26 13:11:56,339] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-26 13:11:56,344] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo , third task']
[2022-07-26 13:11:56,366] {subprocess.py:85} INFO - Output:
[2022-07-26 13:11:56,370] {subprocess.py:92} INFO - , third task
[2022-07-26 13:11:56,372] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-26 13:11:56,436] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=task3, execution_date=20220726T131153, start_date=20220726T131155, end_date=20220726T131156
[2022-07-26 13:11:56,512] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-26 13:11:56,666] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
